styles/github/catppuccin.user.css

```
@@ -13,6 +13,9 @@
 @var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
 @var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
 @var select accentColor "Accent" ["rosewater:Rosewater*", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+
+@var checkbox search-beta "beta: search" 1
+@var checkbox repo-beta "beta: search" 1
```

```suggestion
@var checkbox search-beta "[BETA]: Enable Search" 1
@var checkbox repo-beta "[BETA]: Enable Repo" 1
```

Something in the format of that would be better, and we should actually make clear what those features are. What is "search"? Is that like the command palette? What is "repo"? Is that like the new code view, or the repository overview refresh?

---

That's a good point, i should clarify it more.